operator/pkg/ingress/envoy_config.go

```diff
@@ -442,6 +442,11 @@ func getVirtualHost(ingress *slim_networkingv1.Ingress, rule slim_networkingv1.I
 		routes = append(routes, &route)
 	}
 
+	// This is to make sure that the Exact match is always having higher priority.
+	// Each route entry in the virtual host is checked, in order. If there is a match, the route is used and no further route checks are made.
+	// Related docs https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/route_matching
+	sort.Sort(routes)
```

Comment:
We could use `sort.Slice` instead of having to define the sort.Sort interface on a different file.